x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/action_bar/action_bar.tsx

```diff
@@ -139,11 +146,23 @@ export const ActionBar = ({
             <EuiFlexItem grow={false}>
               <WarningText>{!isValid && hasBeenSubmitted && VALIDATION_ERROR_LABEL}</WarningText>
             </EuiFlexItem>
+
+            <EuiToolTip
+              position="top"
+              content={<p>Works on any kind of element &mdash; buttons, inputs, you name it!</p>}
+            >
+              <EuiButton disabled={true} onClick={() => {}}>
+                Hover me
+              </EuiButton>
+            </EuiToolTip>
```

Comment:
```suggestion
```

Reply:
Oops! Checked-in the example piece. Thanks for spotting.